js/index.js

```diff
@@ -0,0 +1,68 @@
+$(document).ready(function(){
+
+	$('#city-entry').submit(cityEntry);
+	// when form is submitted, run function city entry
+
+	function cityEntry(event) {
+		event.preventDefault();
+		// prevent page from refreshing after entry
+		var city = $('#city-type').val();
+		// store the value of user input from city entry form as variable city
+
+		if (city === 'New York' || city === 'New York City' || city === 'NYC' || city === 'new york' || city === 'new york city' || city === 'nyc') {
```

Comment:
Nice job on capital vs. lowercase - but what if the user input was stored always as lowercase? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase